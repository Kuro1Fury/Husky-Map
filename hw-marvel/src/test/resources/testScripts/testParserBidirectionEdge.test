# Tests the edge constructed by parser is bi-direction

LoadGraph chars twoNodesOneEdge.csv

FindPath chars Mario Luigi

FindPath chars Luigi Mario