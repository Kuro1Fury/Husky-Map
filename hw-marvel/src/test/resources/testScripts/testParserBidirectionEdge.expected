# Tests the edge constructed by parser is bi-direction

loaded graph chars

path from Mario to Luigi:
Mario to Luigi via Super-Mario-Bros

path from Luigi to Mario:
Luigi to Mario via Super-Mario-Bros
